Makefile

```diff
@@ -47,14 +52,13 @@ boshrelease-pipelines: ## Upload boshrelease pipelines to concourse
 app-deployment-pipelines: ## Upload app deployment pipelines to concourse
 	@scripts/build-app-deployment-pipelines.sh
 
-.PHONY: integration-test-pipelines
-integration-test-pipelines: ## Upload integration test pipelines to concourse
-	@scripts/integration-test-pipelines.sh
+.PHONY: git-release-pipelines
```

Comment:
I think we should use the existing `integration-test-pipelines` target and extend the `pipelines/integration-test.yml` pipeline to do tagging because it will make the changes easier to read, rather than removing everything and adding new.

I'm indifferent about whether we rename it to something like `integration-test-tag` to indicate that it now does tagging. If we do rename though, it would be good to do that in a separate commit, again so that history is easier to read.

Could you also squash the "combine pipelines for PR feedback" commit when you do so?